dcc-portal-ui/app/scripts/common/js/location.js

```diff
@@ -138,6 +138,26 @@
         delete s[param];
         $location.search(s);
       },
+      getPaginationParam: function(dataType){
+        var params = {},
```

Comment:
Ditto on using `_.defaults`
Would recommend reading through the documentation on lodash for each function, it will save you a few thousand lines of code over a year or two